app/components/status_page_component.rb

```diff
@@ -12,6 +12,7 @@ class StatusPageComponent < BaseComponent
     ButtonComponent.new(**button_options, big: true, wide: true)
   end
   renders_one :troubleshooting_options, TroubleshootingOptionsComponent
+  renders_one :footer, PageFooterComponent
```

Comment:
Do we need `footer if footer?` while rendering one footer in this component?

Reply:
> Do we need `footer if footer?` while rendering one footer in this component?

Technically no. Counter-intuitively, `renders_one` means it can render 0 or 1 footer. I'd added the condition in case it was necessary for places we don't render a footer. But it looks like ViewComponent handles that internally without the condition.

Reply:
Removed condition in 6695b04e2a with additional spec covering the scenario I was worried about in my previous comment.